pinot-controller/src/test/java/org/apache/pinot/controller/api/PinotTableRestletResourceTest.java

```diff
@@ -536,9 +552,9 @@ public void testValidate()
             .build();
 
     try {
-      ControllerTestUtils
-          .sendPostRequest(StringUtil.join("/", ControllerTestUtils.getControllerBaseApiUrl(), "tables", "validate"),
-              offlineTableConfig.toJsonString());
+      ControllerTestUtils.sendPostRequest(
```

Comment:
would be nice to have couple of tests in validate. 
1. validation success, but unparseable props
2. validation failed 
to make sure both these experiences are unaffected by this change
(optional) test case in add and update too

Reply:
Added unrecognizedProperties tests for add, update and validate table APIs.
Validation failed tests already exist for all APIs.